scripts/chroot.sh

```diff
@@ -30,33 +30,36 @@ if [ -n "${KEY}" ]
 then
     echo "Adding APT key: ${KEY}"
     apt-key add "${KEY}"
+    echo $KEY
 fi
 
 # Add all distro PPAs
-if [ $# -gt 0 ]
+if [ -n "${LIVEBRANCHES}" ] && [-n "${CODENAME}"]
 then
     echo "Enabling repository source"
-    ENABLE_SOURCE=--enable-source
-    for repo in "$@"
-    do
-        if [ "$repo" == "--" ]
-        then
-            echo "Disabling repository source"
-            ENABLE_SOURCE=
-        else
-            echo "Adding repository '$repo'"
-            if [[ "${repo}" == "deb "* ]]
-            then
-                echo "${repo}" >> /etc/apt/sources.list
-                if [ -n "${ENABLE_SOURCE}" ]
-                then
-                    echo "${repo}" | sed 's/^deb /deb-src /' >> /etc/apt/sources.list
-                fi
-            else
-                add-apt-repository ${ENABLE_SOURCE} --yes "${repo}"
-            fi
-        fi
+		for newrepo in $LIVEBRANCHES
+		do
+		    echo "Adding preference for '$newrepo'"
+			  tee "/etc/apt/preferences.d/pop-os-staging-${newrepo//./_}" > /dev/null <<-EOF
+Package: *
+Pin: release o=pop-os-staging-$newrepo
+Pin-Priority: 1002
+EOF
+
+			  echo "Adding repository for '$newrepo'"
+			  LINE="deb http://apt.pop-os.org/staging/$newrepo ${CODENAME} main"
+			  if command -v apt-manage
+			  then
+				    apt-manage add "${LINE}" \
+					       --source-code \
+					       --ident "popdev-${LOCALREMOTE}${newrepo//./-}" \
+					       --name "Pop Development Branch $newrepo"
```

Comment:
`apt-manage` supports a `apt-manage add popdev:branch-name` syntax. I think that would work here, and make it slightly simpler?

Reply:
Ah apparently the `-y` for apt-manage is actually `-t` for some reason. [84e5235](https://github.com/pop-os/iso/pull/318/commits/84e5235e064296dfadd3b0324bee4bc7fcbdc1eb) should work